acceptance/tests/provider/package/rpm_ensure_install_multiversion_package.rb

```diff
@@ -0,0 +1,66 @@
+test_name "rpm ensure held package should preserve version if package is already installed"
+confine :to, :platform => /redhat|centos|el|fedora/
+tag 'audit:low'
+
+require 'puppet/acceptance/common_utils'
+extend Puppet::Acceptance::PackageUtils
+extend Puppet::Acceptance::ManifestUtils
+
+package = "kernel-devel"
+
+agents.each do |agent|
+  initially_installed_versions = []
+
+  teardown do
+    available_versions = on(agent.name, "yum --showduplicates list kernel-devel | sed -e '1,/Available Packages/ d' | awk '{print $2}'").stdout
+    initially_installed_versions.each{ |v|
```

Comment:
use a `do...end` for multiline blocks and a more informative variable name instead of `v`, for example: `version`
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/BlockDelimiters